capitalize_decorator.rb

```diff
@@ -0,0 +1,7 @@
+require_relative 'base_decorator'
+
+class CapitalizeDecorator < BaseDecorator
+  def correct_name
+    super.upcase
```

Comment:
- [ ] We are aiming to `capitalize` ('Emilia') the name here not make it `UPCASE` ('EMILIA'). Kindly fix that.